### PR TITLE
SyncBot: Added instructions regarding the webhook rule

### PR DESCRIPTION
**Pull request by sqweelygig**
Change-Type: patch
See: https://www.flowdock.com/app/rulemotion/r-operations/threads/LPuCARQCDX_qbCb0ZcAaOqt-TEy

---
##### Contributor checklist

- [x] Build output been rebuilt and tested
- [ ] Introduces security considerations
- [ ] Tests are included
- [x] Documentation is added or changed
- [ ] Affects the development, build or deployment processes of the component

##### Reviewer Guidelines
- When submitting a review, please pick:
  - '*Approve*' if this change would be acceptable in the codebase (even if there are minor or cosmetic tweaks that could be improved).
  - '*Request Changes*' if this change would not be acceptable in our codebase (e.g. bugs, changes that will make development harder in future, security/performance issues, etc).
  - '*Comment*' if you don't feel you have enough information to decide either way (e.g. if you have major questions, or you don't understand the context of the change sufficiently to fully review yourself, but want to make a comment)
---

### CODE REVIEW

**Comment by sqweelygig** [comment]
@apostolism does this document the extra steps that I forgot about during the setup of the leads inbox?  If yes, then approve, and that will help ensure it does not get forgotten about next time.

File: docresources/SyncBot/syncbot-readme.md [hunk 1]
```diff
@@ -265,6 +265,10 @@ Errors, where possible, are reported in the originating thread as whispers.
 		* Add the flow mapping to the array
 	* `SYNCBOT_LISTENER_CONSTRUCTORS.front.channelPerInbox`
 		* Add the map of inbox to channel to the array
+1) Add the inbox to the `SyncBot <- Inboxes` rule, settings should be as follow:
+	* When `Inbound message`, `Outbound message`, `Outbound reply`, `Comment` and `Mention`
```
Comment (hedss) [inline docresources/SyncBot/syncbot-readme.md]: I get the gist, but I think the bullet points are confusing as it implies these are three separate things. Am I misunderstanding something, or is this not a test and result order?

Comment (sqweelygig) [reply docresources/SyncBot/syncbot-readme.md]: front

Comment (hedss) [reply docresources/SyncBot/syncbot-readme.md]: Understand that, but if it's going to be written in documentation, possibly this is better:
```
Add an appropriate rule:
WHEN: <blah>
IF <at least one> CONDITION IS MET:
  - <condition one>
  - <condition two>
THEN: Send a Webhook: <blah>
```
Personally, to me, this mimics the UI far better than three bullet points.

**Pull request merged**